lib/rspec/core/formatters.rb

```diff
@@ -163,7 +163,6 @@ def add(formatter_to_use, *paths)
         WARNING
         return
       end
-      @formatters << formatter unless duplicate_formatter_exists?(formatter)
```

Comment:
Now that we no longer mutate `@formatter` after the if/elseif/else/end block, should we remove the `return` on line 164?